test/integration_test/helpers.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  global.expect = require('chai').expect;
```

Comment:
Actually, if you get rid of the createBuffer method, there's really no need for this file. You could add `var expect = require('chai').expect;` to the top of your integration test file, and import the SDK directly in your test as well.